warehouse/templates/email/trusted-publisher-added/body.html

```diff
@@ -28,7 +28,11 @@
   {% trans %}Publisher information{% endtrans %}:
   <ul>
     <li>{% trans %}Publisher name{% endtrans %}: {{ publisher_name }}</li>
-    <li>{% trans %}Publisher specification{% endtrans %}: {{ publisher_spec }}</li>
+    <li>{% trans %}Workflow{% endtrans %}: {{ publisher_workflow }}</li>
+    <li>{% trans %}Repository{% endtrans %}: {{ publisher_repository }}</li>
+    {% if publisher_environment %}
+      <li>{% trans %}Environment{% endtrans %}: {{ publisher_environment }}</li>
+    {% endif %}
```

Comment:
Just to tidy the render a bit:

```suggestion
    {% if publisher_environment %}
    <li>{% trans %}Environment{% endtrans %}: {{ publisher_environment }}</li>
    {% endif %}
```